server/models/bill.coffee

```diff
@@ -13,6 +13,10 @@ module.exports = Bill = cozydb.getModel 'Bill',
     fileId: String
     content: String
     isRefund: Boolean
+    clientId: String # Client number in vendor CRM
```

Comment:
From the data I receive from EDF, Bill are not directly linked to a contract, but to the client. It may be something typical from the SAP datamodel.
But we could add a contractId field later, if the case occurs.